spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -76,19 +76,19 @@
     },
     "binaries_linux": {
         "mirrors": [
-            "https://osf.io/cs6zt/?action=download",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516/spinalcordtoolbox-binaries_linux.tar.gz",
         ],
         "default_location": __bin_dir__,
     },
     "binaries_osx": {
         "mirrors": [
-            "https://osf.io/874cy?action=download",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516/spinalcordtoolbox-binaries_osx.tar.gz",
         ],
         "default_location": __bin_dir__,
     },
     "binaries_win": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/test-release/binaries_win.zip",
+            "https://github.com/spinalcordtoolbox/spinalcordtoolbox-binaries/releases/download/r20220516/spinalcordtoolbox-binaries_windows.tar.gz",
```

Comment:
The binaries in these links were correspond to a more recent version of ANTs. (Bumping the ANTs version was necessary to get ANTs to build on the `windows-2019` GitHub Actions CI Runner. More details in https://github.com/spinalcordtoolbox/build_ANTs/pull/3#discussion_r872663217.)

However, this had the unintended consequence of changing the registration results. 

I'm going to continue to debug in https://github.com/spinalcordtoolbox/build_ANTs/pull/3#discussion_r872663217 to find a way to build ANTs on the `windows-2019` runner without having to bump ANTs.